digdag-core/src/test/java/io/digdag/core/workflow/LoopOperatorFactory.java

```diff
@@ -67,7 +69,7 @@ public TaskResult run()
                         subtask);
             }
 
-            if (params.has("_parallel")) {
+            if (ParallelControl.of(params).isParallel()) {
                 generated.set("_parallel", params.get("_parallel", JsonNode.class));
```

Comment:
We're abstracting accesses to `_parallel` property by moving related implementations to  `ParallelControl`. Why don't you move this direct access to `_parallel` property to the class by adding like `ParallelControl#copyIfNeeded(Config src, Config dst)`?

Reply:
Thank you for pointing out, that's a really good point! Updated: `8b2915e`, `724c1a6`.